pipeline/deploy-stage.yml

```
@@ -0,0 +1,205 @@
+                  
+            # Run health checks
+            - task: Bash@3
+              displayName: 'Health Checks'
+              inputs:
+                targetType: 'inline'
+                script: |
+                  echo "🏥 Running health checks..."
+                  
+                  # Get service URLs
+                  SERVER_URL=$(kubectl get service blog-server --namespace ${{ parameters.kubernetesNamespace }} -o jsonpath='{.spec.clusterIP}:{.spec.ports[0].port}')
+                  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)